import_service/lambdas/importProductsFile.ts

```diff
@@ -0,0 +1,46 @@
+import { APIGatewayProxyEvent, APIGatewayProxyHandler } from 'aws-lambda';
+import { S3Client, PutObjectCommand } from "@aws-sdk/client-s3";
+import { getSignedUrl } from "@aws-sdk/s3-request-presigner";
+
+const BUCKET_NAME = process.env.IMPORT_BUCKET_NAME || "test-bucket";
+const REGION = process.env.AWS_REGION || "us-east-2";
+
+const s3Client = new S3Client({ region: REGION });
+
+export const handler: APIGatewayProxyHandler = async (event: APIGatewayProxyEvent) => {
+  console.log("Incoming request:", JSON.stringify(event, null, 2));
+
+  try {
+    const fileName = event.queryStringParameters?.name;
+
+    if (!fileName) {
+      return {
+        statusCode: 400,
+        headers: { "Access-Control-Allow-Origin": "*" },
+        body: JSON.stringify({ message: "Missing 'name' query parameter" }),
+      };
+    }
+
+    const filePath = `uploaded/${fileName}`;
+    const params = {
+      Bucket: BUCKET_NAME,
+      Key: filePath,
+      ContentType: "text/csv",
+    };
+
+    const signedUrl = await getSignedUrl(s3Client, new PutObjectCommand(params), { expiresIn: 60 });
+
+    return {
+      statusCode: 200,
+      headers: { "Access-Control-Allow-Origin": "*" },
+      body: JSON.stringify({ url: signedUrl }),
```

Comment:
please consider Task 5.2 point 6.
https://github.com/rolling-scopes-school/aws/blob/main/aws-developer/05_integration_with_s3/task.md#task-52

```suggestion
      body: signedUrl,
```